org.uqbar.project.wollok.game/src/org/uqbar/project/wollok/game/gameboard/Gameboard.xtend

```diff
@@ -37,17 +37,18 @@ class Gameboard {
 
 	def void start() {
 		new WollokGDXApplication(new GameboardRendering(this), new GameboardConfiguration(this))
-		Runtime.runtime.addShutdownHook(new Thread([
-			Gdx.app.exit
-		]))
+		Runtime.runtime.addShutdownHook(new Thread[stop])
+	}
+	
+	def stop() {
+		Gdx.app.exit
 	}
 	
 	def void draw(Window window) {
 		// NO UTILIZAR FOREACH PORQUE HAY UN PROBLEMA DE CONCURRENCIA AL MOMENTO DE VACIAR LA LISTA
-		for (var i=0; i < this.listeners.size(); i++){
-			try {
-				this.listeners.get(i).notify(this);
-			} 
+		for (var i=0; i < listeners.size(); i++) {
```

Comment:
Yes, we try something like this time ago and didn't work. Maybe we need try it again.